CHANGELOG.md

```diff
@@ -5,12 +5,17 @@ All notable changes to this project will be documented in this file. See [standa
 ## Unreleased
 
 ### ⚠ BREAKING CHANGES
+* **DataStore/API** category using generated Model classes from Amplify CLI will require the minimum Amplify CLI version of 4.21.4. Current implementations do not expect the field types `embedded(type:)` and `embeddedCollection(of:)` that will break with compile time error. When upgrading Amplify Library, please upgrade Amplify CLI before running `amplify codegen models` to codegen the new `Embeddable` types.
```

Comment:
If this is a breaking change, we need a major version bump.

1. Is it really a breaking change?
1. If so, is it really critical to release a breaking change now?

Reply:
i think this is a breaking change because:
the `customType` is not longer generated and is `embedded(type)` and `embeddedCollection(of)` which means if someone upgrades, and they have a schema which had the embedded type in a model, then the app will not build since 'customType` no longer exists in the ModelFieldType's.

without releasing this change means API with customTypes will not generate the correct selection set and deserialization will fail, currently impacting DataStore as well. The impacted is someone using an embedded type in their model.

Reply:
re-worked on the section regarding our decision to do a non-major version bump